ivy.el

```diff
@@ -3956,6 +3963,24 @@ characters (previous if ARG is negative)."
   (interactive "p")
   (ivy--yank-by #'forward-char arg))
 
+(defvar ivy--pulse-overlay nil
+  "Overlay used to highlight yanked word.")
+
+(defvar ivy--pulse-timer nil
+  "Timer used to dispose of ivy--pulse-overlays.")
+
+(defun ivy--pulse-region (begin end)
+  (unless ivy--pulse-overlay
+    (setq ivy--pulse-overlay (make-overlay 0 0)))
+  (when ivy--pulse-timer
+    (cancel-timer ivy--pulse-timer))
+  (move-overlay ivy--pulse-overlay begin end (current-buffer))
+  (overlay-put ivy--pulse-overlay 'face 'ivy-yanked-word)
+  (setq ivy--pulse-timer (run-at-time 0.5 nil 'ivy--delete-pulse-overlay)))
+
+(defun ivy--delete-pulse-overlay ()
+  (move-overlay ivy--pulse-overlay 0 0))
```

Comment:
Again, zero isn't a valid buffer position. Shouldn't we actually delete the overlay, though, so that it doesn't affect redisplay?

```el
(defun ivy--delete-pulse-overlay ()
  (delete-overlay ivy--pulse-overlay)
  (setq ivy--pulse-overlay nil))
```